pythonAPIClient/camera.py

```diff
@@ -14,8 +32,18 @@ class IPCamera(Camera):
     """
     # TODO: define extra retrieval attributes and constructor of ip camera object
     # replace others with desired field names
-    def __init__(self, others):
-        super(IPCamera, self).__init__(others)
+    def __init__(self,cameraID, type, source, lat, lng, country, state, city, resolution_width
+                , resolution_height, is_active_image, is_active_video, utc_offset, timezone_id
+                , timezone_name, reference_logo, reference_url, ip,port,brand,model,image_path,video_path):
+                self.ip = ip
+                self.port = port
+                self.brand = brand
+                self.model = model
+                self.image_path = image_path
+                self.video_path = video_path
+                super(IPCamera, self).__init__(cameraID, type, source, lat, lng, country, state, city, resolution_width
+                , resolution_height, is_active_image, is_active_video, utc_offset, timezone_id
```

Comment:
I think it would be better if we can change 'type' to something like 'camera_type'. it is not a reserved word and we might not use this builtin function, but I think it is better to avoid shadowing it.

Reply:
I agree with this.